charts/redpanda/testdata/template-cases.golden.txtar

```diff
@@ -4999,11 +4999,9 @@ data:
     schema_registry:
       schema_registry_api:
       - address: 0.0.0.0
-        authentication_method: http_basic
```

Comment:
As noted in the original PR, schema registry doesn't actually support the `authentication_method` key.